src/servlets/bedtime/index.ts

```diff
@@ -118,6 +128,60 @@ const getCollectionMetadata = async (collectionId: number, ownerId: number | nul
   }
 }
 
+type CollectiblesMetadata = {
+  user: any
+} & (
+  | {
+    ethCollectibles: Collectible[]
+    solCollectibles: Collectible[]
+    type: 'gallery'
+  }
+  | {
+    collectible: Collectible | null
+    type: 'detail'
+  }
+)
+
+const getCollectiblesMetadata = async (handle: string, collectibleId?: string): Promise<CollectiblesMetadata> => {
+  const user = await getUserByHandle(handle)
+
+  // Get wallets for user
+  const dpURL = process.env.DP_URL
```

Comment:
Per other comment, let's use the selected discovery node rather than hard coding one

Either by calling a method on libs https://github.com/AudiusProject/general-admission/blob/da0f8f65cb5fbefe04f4ea971e8919651782d364/src/servlets/utils/helpers.ts#L10

or by pulling the direct discovery node off of libs (probably preferable until we do more SDK work)

`audiusLibs.discoveryProvider.discoveryProviderEndpoint`